analyzer/src/environment.rs

```diff
@@ -38,6 +38,9 @@ pub struct Environment {
     /// The source object id of the parent environment, if the environment is nested.
     pub parent: Option<SourceId>,
 
+    /// Whether the environment is directly executable.
+    pub is_script: bool,
```

Comment:
This value has been introduced because this is no longer the case due to how injection work. `has_strict_declaration_order(&self)` has already been updated accordingly.